protos/sources/vega/commands/v1/commands.proto

```diff
@@ -24,6 +24,54 @@ message BatchMarketInstructions {
   repeated OrderAmendment amendments = 2;
   // List of order submissions to be processed sequentially.
   repeated OrderSubmission submissions = 3;
+  // List of stop orders submissions to be processed sequentially.
+  repeated StopOrdersSubmission stop_orders_submissions = 4;
+}
+
+// Stop order submission submits stops orders
+// It is possible to make a single stop order submission by
+// specifying a single direction,
+// or an OCO (One Cancels the Other) stop order submission
+// by specifying a configuration for both direction
+message StopOrdersSubmission {
+  // A stop order which will be trigger
+  // if the price rises above a given trigger price.
+  optional StopOrderSetup rises_above = 1;
+  // A stop order which will be trigger
+  // if the price falls below a given trigger price.
+  optional StopOrderSetup falls_below = 2;
+}
+
+// The price and expiry configuration for a stop order
+message StopOrderSetup {
+  // The order to be submitted once the trigger is breached.
+  OrderSubmission order_submission = 1;
+  // An optional expiry timestamp.
+  optional int64 expires_at = 2;
+  // The strategy to adopt if the expiry time is reached.
+  optional vega.StopOrder.ExpiryStrategy expiry_strategy = 3;
+
+  // The trigger which will need to be breached for the order
+  // to be submitted to the book.
+  oneof trigger {
+    // A fixed price at which the order will be submitted.
+    string price = 100;
+    // A trailing percentage at wich the order will be submitted.
+    string trailing_percent_offset = 101;
+  }
+}
+
+// Cancel a stop order.
+// the following combination are available:
+// an empty object will cancel all stop orders for the party
+// a market id alone, will cancel all stop orders in a market
+// a market id and order id will cancel a specific stop order in a market
+// If the stop order is part of an OCO, both stop order will be cancelled.
+message StopOrdersCancellation {
+  // An optional market id.
+  optional string market_id = 1;
+  // An optional stop_order_id.
```

Comment:
```suggestion
  // Optional market ID.
  optional string market_id = 1;
  // Optional order ID.
```

Is this order ID unique to stopped orders or is it the ID of the order that should be stopped? I've changed it assuming it's the order that should be stopped.